pnpm-lock.yaml

```diff
@@ -265,55 +265,6 @@ importers:
         specifier: ^5.3.3
         version: 5.3.3
 
-  examples/.test/big-router-declaration:
```

Comment:
👀

Reply:
that was deleted previously, unsure how it only propagated now